cmd/acra-keys/keys/generate.go

```diff
@@ -57,6 +55,7 @@ type GenerateKeyParams interface {
 	ZoneID() []byte
 	GenerateNewZone() bool
 	GenerateZoneKeys() bool
+	GenerateZoneSymKey() bool
```

Comment:
We use full word `symmetric` everywhere. So lets use it here too -> `GenerateZoneSymmetricKey`.

Reply:
Got it, let me replace.